tests/virt/node/migration_and_maintenance/test_workload_disruption_migration.py

```
@@ -0,0 +1,292 @@
+    ],
+    indirect=True,
+)
+@pytest.mark.usefixtures("vm_stress_started")
+class TestFedoraWorkloadMigration:
+    @pytest.mark.dependency(name=f"{FEDORA_CLASS_NAME}::migrate_vm")
+    @pytest.mark.polarion("CNV-15225")
+    def test_awd_migration_mode(
+        self,
+        vm_with_hotplug_support,
+        vm_background_process_id,
+        migrated_vm_with_hotplug_support,
```

Ruff (0.15.15)

[warning] Unused method argument: `migrated_vm_with_hotplug_support`

(ARG002)